webapp/app.py

```diff
@@ -56,10 +49,6 @@ def create_app(testing=False):
         talisker.logs.set_global_extra({"service": "snapcraft.io"})
```

Comment:
The two lines above here, where we configure two separate sessions with talisker.... this should be done differently but I guess this is out of scope of this PR.

Ideally we would have a single session that we create here with `talisker.requests.get_session()` and then we pass through to these API libraries.

But this is out of scope from this PR, so we should probably create an issue about it.